Kong/docs.konghq.com
# Updated Mesh 2.0 nav bar

### Summary

Updated the Mesh 2.0 nav bar to point to the /dev shared folder in Kuma. This nav should contain all the existing docs in Kuma that apply to Mesh, in addition to all the current Mesh topics. I didn't need to make any redirects. 
### Reason


### Testing

To test:

- In Netlify, go to the pages in the sidebar to verify that they are all there. Click on the pages to make sure they are directing to the correct page. (Note: I had a few page issues locally because they were renamed in the Kuma repo, not sure if they will also cause issues in Netlify. One of these pages is zoneproxy-auth.md. I re-added the submodule, so that should've fixed this issue.)
- What do you think of the page subsections that are listed in the sidebar? That is the way they are in Kuma, but I think it kind of clutters the sidebar unnecessarily in most cases (I think they could be beneficial for long pages, like in the References section). 
- I also added many unlisted files because Kuma doesn't have them in the sidebar, but some pages still link to those reference pages. Not sure if I did those correctly or if you're able to navigate to those unlisted pages in the Mesh doc Netlify preview.
- Any recommendations for a better icon for "Policies"? I looked through the list that we currently have, and thought this one could work, but it's a lot smaller than the other ones.


https://deploy-preview-4679--kongdocs.netlify.app/mesh/latest/
